docs/recipes/circleci-workflows.md

```diff
@@ -1 +1,75 @@
-# CircleCI 2.0 workflows
+# Using semantic-release with [CircleCI 2.0 workflows](https://circleci.com/docs/2.0/workflows/)
+
+![CircleCI workflows screenshot](https://github.com/circleci/circleci-docs/raw/928cbb042453ae182996ce05f4e4042b02a24634/jekyll/assets/img/docs/workflow_detail.png)
+
+Similar to Travis build stages, CircleCI 2.0 workflows are a great way to run semantic-release in an independent build step.
+We can let CircleCI take care of only running a release on master, after tests passed and running it on an image with Node 8 - your test jobs could use a completely different environment, your project may not even be written in JavaScript.
+
+Make sure to set `NPM_TOKEN` and `GH_TOKEN` in Circle's project settings UI.
+
+### Annotated example `.circleci/config.yml`
+
+```yml
```

Comment:
Can this configuration be simplified to keep only what's required by semantic-release, similarly to [travis.md](https://github.com/semantic-release/semantic-release/blob/caribou/docs/recipes/travis.md)?

You can mention it's a minimal configuration for the semantic-release part only and provide a link to the Circle CI doc for users can refer to it for the non semantic-release related parts.

Reply:
I am not sure what you would want it to look like - only have a release step? Imo in opposite to documentation, recipes should be examples that can be copy+pasted and then modified if needed. They should examples show how the general documentation can be adapted for specific projects, more as inspiration than a guarantee to work for every project.

Reply:
I would leave out anything that is not necessary to make semantic-release work. Please leave out all the steps related to caching. I’m not familiar with CircleCI so I don’t know how we could reduce it further, but the simpler it is, the less error prone, the better.